PolyEngine/Engine/Src/KeyBindings.hpp

```diff
@@ -1,11 +1,12 @@
 #pragma once
 
+#if defined(_WIN32)
 #include <Windows.h>
 
 namespace Poly
 {
-	enum class eKey
-	{	
+	enum class eKey : uint8_t
```

Comment:
Why do we want to have base type of uint8_t? This introduces great risk of overflow.